### PR TITLE
clear port counters and jumbo packet, ether statistics

### PR DESCRIPTION
**Pull request by Rajasekhar-Pulluru**

### CODE REVIEW

File: sai/inc/saiport.h [hunk 1]
```diff
@@ -494,12 +497,24 @@ typedef enum _sai_port_stat_counter_t
...
     SAI_PORT_STAT_ETHER_OUT_PKTS_512_TO_1023_OCTETS,
     SAI_PORT_STAT_ETHER_OUT_PKTS_1024_TO_1518_OCTETS,
+    SAI_PORT_STAT_ETHER_OUT_PKTS_1519_TO_2047_OCTETS,
+    SAI_PORT_STAT_ETHER_OUT_PKTS_2048_TO_4095_OCTETS,
+    SAI_PORT_STAT_ETHER_OUT_PKTS_4096_TO_9216_OCTETS,
+    SAI_PORT_STAT_ETHER_OUT_PKTS_9217_TO_16383_OCTETS,
+    SAI_PORT_STAT_ETHER_PKTS_1519_TO_2047_OCTETS,
+    SAI_PORT_STAT_ETHER_PKTS_2048_TO_4095_OCTETS,
+    SAI_PORT_STAT_ETHER_PKTS_4096_TO_9216_OCTETS,
+    SAI_PORT_STAT_ETHER_PKTS_9217_TO_16383_OCTETS,
```
Comment (lguohan) [inline sai/inc/saiport.h]: We already have IN and OUT, what are these counters? 

514 +    SAI_PORT_STAT_ETHER_PKTS_1519_TO_2047_OCTETS,  
 515  +    SAI_PORT_STAT_ETHER_PKTS_2048_TO_4095_OCTETS,  
 516  +    SAI_PORT_STAT_ETHER_PKTS_4096_TO_9216_OCTETS,  
 517  +    SAI_PORT_STAT_ETHER_PKTS_9217_TO_16383_OCTETS,

File: sai/inc/saiport.h [hunk 2]
```diff
@@ -393,6 +393,9 @@ typedef enum _sai_port_attr_t
      * Default no limits */
     SAI_PORT_ATTR_QOS_SCHEDULER_PROFILE_ID,
 
+    /* Clear all statistics counter for a given port [sai_object_id_t] */
+    SAI_PORT_ATTR_CLEAR_ALL_STAT_COUNTER,
```
Comment (lguohan) [inline sai/inc/saiport.h]: attribute is not an action. attribute is something we can set and get. This is something you can only set. Maybe we can have a clear_port_all_stats()

Comment (arunsubash) [inline sai/inc/saiport.h]: Yes we will add an API instead

File: sai/inc/saiport.h (hunk 1 above)
Comment (arunsubash) [inline sai/inc/saiport.h]: This was added to be consistent with the other counters with has all 3 - IN/OUT and both

Comment (lguohan) [inline sai/inc/saiport.h]: for the packet size counter less than 1519, there are not ETHER_PKTS counters. What's this ETHER_PKTS counter, is it the sum of both in and out?

Comment (arunsubash) [inline sai/inc/saiport.h]: SAI_PORT_STAT_ETHER_STATS_PKTS_64_OCTETS,
    SAI_PORT_STAT_ETHER_STATS_PKTS_65_TO_127_OCTETS,
    SAI_PORT_STAT_ETHER_STATS_PKTS_128_TO_255_OCTETS,
    SAI_PORT_STAT_ETHER_STATS_PKTS_256_TO_511_OCTETS,
    SAI_PORT_STAT_ETHER_STATS_PKTS_512_TO_1023_OCTETS,
    SAI_PORT_STAT_ETHER_STATS_PKTS_1024_TO_1518_OCTETS,

Its there in sai_port_stat_counter_

**Comment by lguohan** [comment]
merged, thanks.

**Pull request closed without merging**